lib/locales/de.js

```diff
@@ -11,7 +11,68 @@
+        'Zoom': 'Zoom',                                                  // components/modebar/buttons.js:85
+        'Zoom in': 'Hineinzoomen',                                               // components/modebar/buttons.js:121
+        'Zoom out': 'Herauszoomen',                                              // components/modebar/buttons.js:130
+        'close: ': 'schließen',                                                // traces/ohlc/transform.js:139
+        'trace': 'Datenspur',                                                 // plots/plots.js:439
+        'lat: ': 'Lat.: ',                                                   // traces/scattergeo/calc.js:48
+        'lon: ': 'Lon.: ',                                                  // traces/scattergeo/calc.js:49
+        'q1: ': 'q1',                                                    // traces/box/calc.js:130
+        'q3: ': 'q3',                                                    // traces/box/calc.js:131
+        'source: ': 'Quelle: ',                                                // traces/sankey/plot.js:140
+        'target: ': 'Ziel: ',                                          // traces/sankey/plot.js:141
+        'lower fence: ': 'Untere Schranke',                                           // traces/box/calc.js:134
+        'upper fence: ': 'Obere Schranke: ',                                           // traces/box/calc.js:135
+        'max: ': 'Max.: ',                                                   // traces/box/calc.js:132
+        'mean ± σ: ': 'Mittelwert ± σ: ',                                            // traces/box/calc.js:133
+        'mean: ': 'Mittelwert: ',                                                  // traces/box/calc.js:133
+        'median: ': 'Median: ',                                                // traces/box/calc.js:128
+        'min: ': 'Min.: ',                                                 // traces/box/calc.js:129
+
+        // Missing context
+        'open: ': 'Offen: ',                                                  // traces/ohlc/transform.js:136
+        'high: ': '',                                                  // traces/ohlc/transform.js:137
+        'low: ': '',                                                   // traces/ohlc/transform.js:138
+        'kde: ': '',                                                   // traces/violin/calc.js:73
+        'outgoing flow count: ': '',                                   // traces/sankey/plot.js:143
+        'incoming flow count: ': '',                                   // traces/sankey/plot.js:142
+
+        // Missing translation
+        'Toggle Spike Lines': '',                                      // components/modebar/buttons.js:548
+        'Toggle show closest data on hover': '',                      // components/modebar/buttons.js:353
+        'Turntable rotation': ''                                     // components/modebar/buttons.js:288
```

Comment:
`Toggle Spike Lines`: spike lines are what we call the lines to the axis or across the plot that can show up when you hover on a point - this button turns them on or off. See original PR https://github.com/plotly/plotly.js/pull/1461

`Toggle show closest data on hover`: sometimes there are two buttons available: "show closest data on hover" and "compare data on hover". Other times there is only one button, because only "closest" is meaningful, and this button turns it on or off (no hover effects at all).

`Turntable rotation`: This is a funny one: in 3D plots, you have two different rotation modes: "Orbital rotation" (translated above as "Orbitalrotation") allows you to grab the scene and rotate it in a path-dependent way along any rotation axis (ie at any instant the rotation axis is perpendicular to both the 3-vector you started at, and the 3-vector implied by your mouse motion. "Turntable rotation" maps x motion of the mouse to rotation around the z axis, and y motion of the mouse to tilt of the z axis toward or away from the viewer. This does not allow you to tip the scene sideways but can be more intuitive to use. It is named after the old record player turntables.

Reply:
I know what it does, but I'm not aware of the German word. 

The closest translation I've got in mind is "Fadenkreuz" (engl. crosshairs). But I think it is a bad translation. I'm sure there is a better translation.

Reply:
Blender calls turntable rotation "Drehscheibenorbit"... not sure if this is a commonly understood term, though

Reply:
I don't think orbital / turntable rotation is particularly commonly understood in English either ;) As long as it's clear to users that these are two different rotation modes, I'm happy.